clipboard-apis/async-idlharness.https.window.js

```diff
@@ -0,0 +1,17 @@
+// META: timeout=long
```

Comment:
Can you rename this to just `idlharness.https.window.js`? There's no other idlharness.js test to distinguish this from, and this doesn't do anything async beyond the implicit fetching.